code/go/0chain.net/chaincore/transaction/handler.go

```diff
@@ -29,24 +29,21 @@ func PutTransaction(ctx context.Context, entity datastore.Entity) (interface{},
 		return nil, fmt.Errorf("invalid request %T", entity)
 	}
 
-	debugTxn := txn.DebugTxn()
+	//debugTxn := txn.DebugTxn()
 	err := txn.Validate(ctx)
 	if err != nil {
 		logging.Logger.Error("put transaction error", zap.String("txn", txn.Hash), zap.Error(err))
 		return nil, err
 	}
-	if debugTxn {
-		logging.Logger.Info("put transaction (debug transaction)", zap.String("txn", txn.Hash), zap.String("txn_obj", datastore.ToJSON(txn).String()))
-	}
+	//if debugTxn {
```

Comment:
uncomment it please

Reply:
We will log the txn hash to see if the txn is submitted to the miner, but will remove the transaction object.